src/controllers/LoginController.ts

```diff
@@ -36,11 +42,35 @@ export default class LoginController {
     if (!isPasswordValid) {
       return res.status(401).json({ message: 'Invalid email or password' })
     }
-
+    LoginController.loggedInUserId = existingUser.id
     const token = generateToken({
       id: existingUser.id,
       email: existingUser.email,
     })
+    redisClient.setEx(`user:${existingUser.id}`, 24 * 60 * 60, token)
```

Comment:
```suggestion
import {JWT_EXPIRE_TIME} from "../config"
    redisClient.setEx(`user:${existingUser.id}`, JWT_EXPIRE_TIME, token)
```
it would be better to use expire time that is same as that of JWT